### ISSUE TITLE
what is the most stable version for coq 8.12, 0.11.7 or dev? is dev stable?

### ISSUE THREAD

**Issue by brando90** [coq-community/coq-ext-lib#132]
I want to be able to pin forever for a version of coq 8.12. Currently this works but worried
```
opam install -y coq-ext-lib
opam install -y coq-ext-lib=dev  # dev versions always make me unease
opam install -y coq-ext-lib=0.11.7  # this seemed to work but it says it downgraded, for now leave as is
```
what do developers recommend? Thanks!

**Comment by Lysxia** [comment]
We recommend the most recently released version that is compatible with Coq 8.12, which is just the latest version 0.11.7.

I think as a general rule of thumb you should just try `opam install package` and if that succeeds, note the version, pin it, and voilà.

The message you got was probably saying that it downgraded from dev. `dev` is by definition not stable, it usually means "whatever you get by `git clone`", just nicely packaged.

**Comment by liyishuai** [comment]
Propose closing as completed, if no further concerns arise this week.
